fetch.bs

```diff
@@ -6710,6 +6748,364 @@ agent's <a>CORS-preflight cache</a> for which there is a <a>cache entry match</a
+
+<p>In the following sequence, the first two requests would succeed, but the third one would throw.
+That's because al overall 640 kibibytes quota was not exceeded in the first to call, however the 3rd
+request exceeds the reporting-origin quota for <code>https://a.example.com</code>, and would throw.
+<pre><code class=lang-javascript>
+  fetchLater("https://a.example.com", {method: "POST", body: a_64kb_body});
+  fetchLater("https://b.example.com", {method: "POST", body: a_64kb_body});
+  fetchLater("https://a.example.com");
+</code></pre>
+
+<p>Same-origin same-agent subframes share the quota of their parent. However, cross-origin or
+cross-agent iframes only receive 8kb of quota by default. So in the following example, the first 3
+calls would succeed and the last one would throw.
+<pre><code class=lang-javascript>
+  // In main page
+  fetchLater("https://a.example.com", {method: "POST", body: a_64kb_body});
+
+  // In same-origin iframe
+  fetchLater("https://b.example.com", {method: "POST", body: a_64kb_body});
+
+  // In cross-origin iframe at https://frame.example.com
+  fetchLater("https://a.example.com", {body: a_5kb_body});
+  fetchLater("https://a.example.com", {body: a_12kb_body});
+</code></pre>
+
+<p>To make the previous example not throw, the top-level {{Document}} needs to delegate some of its
+quota to <code>https://frame.example.com</code>, for example by serving the following header:
+<pre><code>Permissions-Policy: deferred-fetch=(self "https://frame.example.com")</code></pre>
+
+</div>
+
+
+<p>This specification defined a <a>policy-controlled feature</a> identified by the string
+<dfn for=PermissionPolicy enum-value>"deferred-fetch"</dfn>. Its
+<a for="policy-controlled feature">default allowlist</a> is "<code>self</code>".
+
+<p>This specification defined a <a>policy-controlled feature</a> identified by the string
+<dfn for=PermissionPolicy enum-value>"deferred-fetch-minimal"</dfn>. Its
+<a for="policy-controlled feature">default allowlist</a> is "<code>*</code>".
+
+<p>The <dfn>optional subframe deferred-fetch quota</dfn> is 64 kibibytes.
+<p>The <dfn>minimal subframe deferred-fetch quota</dfn> is 8 kibibytes.
+<p>The <dfn>deferred-fetch delegated quota</dfn> is 128 kibibytes.
+
+<div algorithm>
+<p>Two {{Document}}s <var>a</var> and <var>b</var> are said to
+<dfn>share deferred-fetch quota</dfn> if <var>a</var>'s
+<a>relevant agent</a> is <var>b</var>'s <a>relevant agent</a>, and <var>a</var>'s
+<a for=Document>URL</a>'s <a for=url>origin</a> is <a>same origin</a> with <var>b</var>'s
+<a for=Document>URL</a>'s <a for=url>origin</a>.
+</div>
+
+
+<div algorithm>
+<p>To get the <dfn>available deferred-fetch quota</dfn> given a {{Document}}
+<var>document</var> and an <a for=/>origin</a>-or-null <var>origin</var>:
+
+<ol>
+ <li><p>Let <var>quota</var> be zero.
+
+ <li><p>Let <var>quotaForOrigin</var> be 64 kibibytes.
+
+ <li>
+  <p>For each <var>otherNavigable</var> in <var>document</var>'s <a>node navigable</a>'s
```

Comment:
```
For each otherNavigable in document’s node navigable’s top-level traversable’s inclusive descendant navigables:
```

* Does this iterate through all dom elements? If so, `otherNavigable` can be non-container elements?